Source/RuntimeMeshComponent/Public/RuntimeMeshCollision.h

```diff
@@ -19,31 +19,31 @@ struct RUNTIMEMESHCOMPONENT_API FRuntimeMeshCollisionConvexMesh
 
 public:
 	FRuntimeMeshCollisionConvexMesh() : BoundingBox(ForceInit) { }
-	FRuntimeMeshCollisionConvexMesh(const TArray<FVector>& InVertexBuffer)
+	FRuntimeMeshCollisionConvexMesh(const TArray<FVector3f>& InVertexBuffer)
 		: VertexBuffer(InVertexBuffer)
 		, BoundingBox(InVertexBuffer)
 	{
 	}
-	FRuntimeMeshCollisionConvexMesh(TArray<FVector>&& InVertexBuffer)
+	FRuntimeMeshCollisionConvexMesh(TArray<FVector3f>&& InVertexBuffer)
 		: VertexBuffer(InVertexBuffer)
 		, BoundingBox(VertexBuffer)
 	{
 	}
-	FRuntimeMeshCollisionConvexMesh(const TArray<FVector>& InVertexBuffer, const FBox& InBoundingBox)
+	FRuntimeMeshCollisionConvexMesh(const TArray<FVector3f>& InVertexBuffer, const FBox3f& InBoundingBox)
 		: VertexBuffer(InVertexBuffer)
 		, BoundingBox(InBoundingBox)
 	{
 	}
-	FRuntimeMeshCollisionConvexMesh(TArray<FVector>&& InVertexBuffer, const FBox& InBoundingBox)
+	FRuntimeMeshCollisionConvexMesh(TArray<FVector3f>&& InVertexBuffer, const FBox3f& InBoundingBox)
 		: VertexBuffer(InVertexBuffer)
 		, BoundingBox(VertexBuffer)
 	{
 	}
 
 	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "RuntimeMesh|Collision|Convex")
-	TArray<FVector> VertexBuffer;
-	UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "RuntimeMesh|Collision|Convex")
-	FBox BoundingBox;
+	TArray<FVector3f> VertexBuffer;
+	//UPROPERTY(EditAnywhere, BlueprintReadWrite, Category = "RuntimeMesh|Collision|Convex")
+	FBox3f BoundingBox;
```

Comment:
Also, `FRuntimeMeshCollisionConvexMesh` member `FBox BoundingBox` changed to `FBox3f BoundingBox` and is no longer `UPROPERTY` (`FBox3f` not a supported `UPROPERTY` type). This may be possible to change back by computing the bounding box in the constructor by first making a `FBox3f` from the `TArray<FVector3f>& InVertexBuffer` then converting to double precision `FBox`.

Reply:
This is the only place where `BoundingBox` is directly referenced: https://github.com/TriAxis-Games/RuntimeMeshComponent/pull/223/files#diff-661d54bf3cc832a489088960f4bc6edd8913b26707d8290818b2ca52277e9ebdR950-R956